### ISSUE TITLE
[Bug]: @remix-run/serve is missing

### ISSUE THREAD

**Issue by estruyf** [remix-run/remix#681]
### Which Remix packages are impacted?

- [ ] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [ ] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [ ] `@remix-run/react`
- [X] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

1.0.5

### Steps to Reproduce

Created a new project with `npx create-remix@latest`. I choose to host it on Vercel + Typescript. In the end, I did not install the dependencies.

When I opened my terminal to the folder, manually ran an npm installation and ran `npm run dev`. I got the following error:

```
Could not locate @remix-run/serve. Please verify you have it installed to use the dev command.
```

### Expected Behavior

Would expect the `@remix-run/serve` to be included as a developer dependency.

### Actual Behavior

`@remix-run/serve` is not referenced as a dev. dependency.

**Comment by mcansh** [comment]
yep, sorry about that, fixed in [1.0.6](https://github.com/remix-run/remix/releases/tag/v1.0.6)

**Comment by estruyf** [comment]
Great!!!
